Formula/git-remote-hg.rb

```diff
@@ -17,7 +17,8 @@ class GitRemoteHg < Formula
 
   # Requires Python2.
   # https://github.com/Homebrew/homebrew-core/issues/93940
-  deprecate! date: "2022-04-23", because: :unsupported
+  # Original deprecation date: 2022-04-23
+  disable! date: "2022-11-03", because: :unsupported
```

Comment:
Has anyone tried git-remote-hg 0.6?

Reply:
I did notice it after opening PR and was looking into it:
```diff
diff --git a/Formula/git-remote-hg.rb b/Formula/git-remote-hg.rb
index 32b962c48ba..734cb68b592 100644
--- a/Formula/git-remote-hg.rb
+++ b/Formula/git-remote-hg.rb
@@ -1,12 +1,11 @@
 class GitRemoteHg < Formula
-  include Language::Python::Virtualenv
+  include Language::Python::Shebang
 
   desc "Transparent bidirectional bridge between Git and Mercurial"
   homepage "https://github.com/felipec/git-remote-hg"
-  url "https://github.com/felipec/git-remote-hg/archive/v0.4.tar.gz"
-  sha256 "916072d134cde65b7ffa7d1da1acaabb0f29b65c017d0560e907e7a94063d1b1"
+  url "https://github.com/felipec/git-remote-hg/archive/refs/tags/v0.6.tar.gz"
+  sha256 "1d49ffda290c8a307d32191655bdd85015e0e2f68bb2d64cddea04d8ae50a4bf"
   license "GPL-2.0"
-  revision 2
   head "https://github.com/felipec/git-remote-hg.git", branch: "master"
 
   bottle do
@@ -15,30 +14,17 @@ class GitRemoteHg < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "1380e5053a25462f27d9be329840b6dda55b08e01b70ed6c581f3c625c7b332d"
   end
 
-  # Requires Python2.
-  # https://github.com/Homebrew/homebrew-core/issues/93940
-  deprecate! date: "2022-04-23", because: :unsupported
-
-  depends_on "asciidoc" => :build
-  depends_on :macos # Due to Python 2
-
-  uses_from_macos "libxml2"
-  uses_from_macos "libxslt"
+  depends_on "asciidoctor" => :build
+  depends_on "mercurial"
+  depends_on "python@3.10"
 
   conflicts_with "git-cinnabar", because: "both install `git-remote-hg` binaries"
 
-  resource "hg" do
-    url "https://www.mercurial-scm.org/release/mercurial-5.2.2.tar.gz"
-    sha256 "ffc5ff47488c7b5dae6ead3d99f08ef469500d6567592a25311838320106c03b"
-  end
-
   def install
-    venv = virtualenv_create(libexec)
-    venv.pip_install resource("hg")
-    inreplace "git-remote-hg", /#!.*/, "#!#{libexec}/bin/python"
+    rewrite_shebang detected_python_shebang, "git-remote-hg"
     system "make", "install", "prefix=#{prefix}"
 
-    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
+    ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
     system "make", "install-doc", "prefix=#{prefix}"
   end
 ```

Reply:
Maybe worth dropping the disable in this PR then and, if needed, open a separate PR to do it.

Reply:
Created PR above diff: #114635